source/overview/security.rst

```diff
@@ -2,112 +2,112 @@
 
-Centralized Security and Administration 
+Centralized Security and Administration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
    - Manage users, teams, access control and system settings in a web-based `System Console user interface <https://docs.mattermost.com/administration/config-settings.html>`_.
 
-Transmission Security 
+Transmission Security
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions across both LAN and internet. 
+   - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions across both LAN and internet.
    - Connections to Active Directory/LDAP can be optionally secured with TLS or stunnel (E10).
-   - Encryption-at-rest is available through hardware and software disk encryption solutions applied to the Mattermost database, which can reside on its own server within your infrastructure.  
+   - Encryption-at-rest is available through hardware and software disk encryption solutions applied to the Mattermost database, which can reside on its own server within your infrastructure.
    - Option to `exclude message contents from push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ to comply with strict compliance policies, such as US HIPAA standards.
-   - Ability to exclude or include the `contents of messages in push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ to avoid disclosure on locked mobile screens, and via relay servers from Apple and Google when sending notifications to iOS or Android mobile apps (relevant to compliance standards such as HIPAA) 
-   
+   - Ability to exclude or include the `contents of messages in push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`_ to avoid disclosure on locked mobile screens, and via relay servers from Apple and Google when sending notifications to iOS or Android mobile apps (relevant to compliance standards such as HIPAA)
+
 Integrity & Audit Controls
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   - By default, Mattermost stores a complete history of messages, including edits and deletes, along with all files uploaded. User interface actions for "deleting" messages, channels and private groups only remove the data from the user interface, the data is retained within your database. 
+   - By default, Mattermost stores a complete history of messages, including edits and deletes, along with all files uploaded. User interface actions for "deleting" messages and channels only remove the data from the user interface, the data is retained within your database.
```

Comment:
"...from the user interface, the data is retained within your database." > "...from the user interface. The data is retained within your database."